code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -36,6 +36,15 @@
 	can_pull_size = TINY_ITEM
 	can_pull_mobs = MOB_PULL_NONE
 
+/mob/living/simple_animal/mouse/New()
+	if(prob(2))

```

Comment:
You need to call `..()` when overriding `New()`. `..()` is the DM equivalent of [super](https://stackoverflow.com/questions/5141344/meaning-of-super-keyword).